CRM/CiviAwards/Setup/DeleteApplicantReviewCustomField.php

```diff
@@ -6,9 +6,17 @@
 class CRM_CiviAwards_Setup_DeleteApplicantReviewCustomField {
 
   /**
-   * Deletes the Applicant review custom group set.
+   * Removes the Applicant review activity group and statuses.
    */
   public function apply() {
+    $this->removeApplicantReviewCustomGroup();
```

Comment:
I don't think we need the cleanup part for the Applicant review statuses since we are not deleting the applicant review activity type. Besides this is also a wrong place to add logic like this since this class names describes that it deletes custom fields.

Here is what I will suggest:

- Let's revert this file back to what it was.
- Let's make the setApplicantReviewActivityTypeStatuses in the `CRM_CiviAwards_Setup_CreateApplicantReviewActivityType` to be idempotent, i.e we check if the `Applicant Review ` is already part of the grouping for the concerned statuses, if it is, we don't bother adding it. 

That way if the extension is un-installed and re-installed, we won't have duplicate data